### DISCUSSION TITLE
怎么办

### DISCUSSION THREAD

**Discussion by wshtdgo** [share121/inter-knot#5278] [委托]
感觉现在根本对原神提不起兴趣去玩，怎么办，我感觉我号要废了，谁能帮我提一提兴趣（简而言之就是能不能有个大佬带带我）本人世界等级八，冒险等级55，uid271870648，我是真的不想一个人玩呜呜呜

**Comment by KELERSAMA** [comment]
去带16级大佬呱
<img src="https://bbs-static.miyoushe.com/static/2023/05/23/a2fa805a0a782cdb8ab44ddaf8d5343d_962898009752203374.png" alt="阿姬倒地了04" width="75" >

**Comment by Kazuha-hhh** [comment]
连歪好几次，想提兴趣也不上来，好友列表也几乎都是31天前登录的

**Comment by YeYingMakas** [comment]
是的，我也觉得无聊了，不怎么玩了，最近纳塔开了回来打了打，主要还是太费时间了，忙起来也没空打

**Comment by manyshoes** [comment]
也没必要把大片时间都花在上面，这段时间没兴趣了就放下。我有些版本都是只过主线或者直接a了，什么时候有心思了看着补一下

**Comment by nabeilishi** [comment]
uid 183182089 来原摄吧

**Comment by NanaChan962** [comment]
原神真玩不下去一点了，zzz也长草了，就崩铁模拟宇宙玩着有点意思

**Comment by Psinary** [comment]
去带萌新吧，一堆萌新嗷嗷待哺<img src="https://img-static.mihoyo.com/communityweb/upload/9d2cd2497aa2c935ed084b2d539b830a.png" alt="阿贝多-坏笑" width="75" >

**Comment by Loxia1121** [comment]
把原神当成看番工具（）无聊就打打深渊看看老公老婆们的邀约和传说 拍拍照 看看壶 刷刷圣遗物
时间就这样过去了（bushi

**Reply by Tu-Doou** [reply]
说的太对了

**Comment by TNCNET** [comment]
寄...顺便我是官服的，101058771

另外记得钟离的一句话「旅程不必匆忙」。原本身比较开放（确信），玩法也不限于这类基础的东西。还有摄影；壶内建筑设计；带带萌新。一堆<45级的都可以算（确信）；打牌；发呆看看风景；潜水（枫丹地区）...嗯？法子很多了。别把思路局限在刷遗物这些基础操作之中。

**Comment by Corvaine** [comment]
没有兴趣了就干脆退游一阵子
_(:з」∠)_

**Reply by TNCNET** [reply]
也是一个好主意
我不知道上哪能带个萌新（
